src/main/java/com/google/cloud/discotoproto3converter/proto3/Proto3Writer.java

```diff
@@ -194,4 +196,16 @@ private String formatDescription(String prefix, String description) {
     comments = comments.replaceAll(" *\n", "\n");
     return prefix + "// " + comments;
   }
+
+  private void writeLicenseAndWarning(PrintWriter writer, ProtoFile protoFile) {
+    writer.println(protoFile.getLicense());
+
+    writer.println("// Generated by the disco-to-proto3-converter. DO NOT EDIT!");
```

Comment:
Consider having the `DO NOT EDIT` at the start of the line for increased visibility

Reply:
I was following the same format as protoc does for protobuf stubs. 

This is what protoc puts at the top of the generated stubs:

```
// Generated by the protocol buffer compiler.  DO NOT EDIT!
// source: google/bigtable/admin/v2/instance.proto
```